config/config_test.go

```diff
@@ -93,37 +94,47 @@ func TestFromConfig(t *testing.T) {
 	grpcProxy := fiber.NewProxy(nil, grpcCaller)
 
 	tests := []struct {
-		name       string
-		configPath string
-		want       fiber.Component
+		name              string
+		configPath        string
+		expectedComponent fiber.Component
+		expectedErrMsg    string
 	}{
 		{
-			name:       "http proxy",
-			configPath: "../internal/testdata/config/http_proxy.yaml",
-			want:       httpProxy,
+			name:              "http proxy",
+			configPath:        "../internal/testdata/config/http_proxy.yaml",
+			expectedComponent: httpProxy,
 		},
 		{
-			name:       "grpc proxy",
-			configPath: "../internal/testdata/config/grpc_proxy.yaml",
-			want:       grpcProxy,
+			name:              "grpc proxy",
+			configPath:        "../internal/testdata/config/grpc_proxy.yaml",
+			expectedComponent: grpcProxy,
+		},
+		{
+			name:           "grpc proxy",
+			configPath:     "../internal/testdata/config/invalid_grpc_proxy.yaml",
+			expectedErrMsg: "fiber: grpc dispatcher: missing config (endpoint/service/method/response-proto)",
```

Comment:
We removed the `response-proto` from the config, right? The error message from `NewDispatcher` should be updated.